en/advanced_features/traffic_avoidance_utm.md

```diff
@@ -0,0 +1,34 @@
+# Air Traffic Avoidance: MAVLink
+
+PX4 can use MAVLink messages to support simple air traffic avoidance in [missions](../flight_modes/mission.md).
+If a potential collision is detected, PX4 can *warn*, immediately [land](../flight_modes/land.md), or [return](../flight_modes/return.md) (depending on the value of [NAV_TRAFF_AVOID](#NAV_TRAFF_AVOID)).
+
+This implementation is based on the [ADS-B traffic avoidance](../advanced_features/traffic_avoidance_adsb.md). The `UTM_GLOBAL_POSITION` MAVLink
+message is treated exactly the same as an ADS-B message.
+
+
+## Configure Traffic Avoidance
+
+Configure the action when there is a potential collision using the parameter below:
+
+Parameter | Description
+--- | ---
+<span id="NAV_TRAFF_AVOID"></span>[NAV_TRAFF_AVOID](../advanced_config/parameter_reference.md#NAV_TRAFF_AVOID) | Enable traffic avoidance mode specify avoidance response. 0: Disable, 1: Warn only, 2: Return mode, 3: Land mode.
+
+
+## Implementation
+
+PX4 listens for valid `UTM_GLOBAL_POSITION` MAVLink messages during missions. When one is received, it is converted
+into a transponder report in the format used by [ADS-B traffic avoidance](../advanced_features/traffic_avoidance_adsb.md).
+
+If a valid transponder report is received, PX4 first uses the transponder position and heading information to estimate whether the vehicles will share a similar altitude before they pass each other.
```

Comment:
Since behaviour is similar for both cases, I suggest that for now we just link to the other docs. Then here we explain the sacrifices/differences.

So, how about:

PX4 listens for `UTM_GLOBAL_POSITION` MAVLink messages during missions.
When a valid message is received, its validity flags, position and heading are mapped into the same transponder report UORB topic used for *ADS-B traffic avoidance*.

The implementation is otherwise *exactly* as described in: [ADS-B traffic avoidance > Implementation](../advanced_features/traffic_avoidance_adsb.md#implementation).

> **Note** [UTM_GLOBAL_POSITION](https://mavlink.io/en/messages/common.html#UTM_GLOBAL_POSITION) contains additional fields that are not provided by an ADBS transponder (see [ADSB_VEHICLE](https://mavlink.io/en/messages/common.html#ADSB_VEHICLE)).
  The current implementation simply drops the additional fields (including information about the vehicle's planned next waypoint).